helm/kiam-app/values.yaml

```diff
@@ -135,3 +135,16 @@ namespaceAnnotator:
     requests:
       cpu: 50m
       memory: 75Mi
+
+# set the HTTP_PROXY, HTTPS_PROXY and NO_PROXY variable
+proxy:
+  noProxy:
```

Comment:
I'm not a fan of this empty syntax. It is easy to get it wrong by changing indentation of any of these fields. How do you feel about specifying a default value, such as `""` ?

Reply:
than the condition wont work, maybe `nil` would work

Reply:
`null` value works

Reply:
better?

Reply:
yeah totally thank you <3